src-db/database/sourcedata/AD_MESSAGE.xml

```diff
@@ -23755,6 +23755,18 @@ The column has been set in the Dimension list to filter a Sequence.]]></MSGTEXT>
 <!--9E7F116DBB424386BB82CCD76514D4EE-->  <ISINCLUDEINI18N><![CDATA[N]]></ISINCLUDEINI18N>
 <!--9E7F116DBB424386BB82CCD76514D4EE--></AD_MESSAGE>
 
+<!--9EEEEB51E748402DBB3C41DC04CFA16C--><AD_MESSAGE>
+<!--9EEEEB51E748402DBB3C41DC04CFA16C-->  <AD_MESSAGE_ID><![CDATA[9EEEEB51E748402DBB3C41DC04CFA16C]]></AD_MESSAGE_ID>
+<!--9EEEEB51E748402DBB3C41DC04CFA16C-->  <AD_CLIENT_ID><![CDATA[0]]></AD_CLIENT_ID>
+<!--9EEEEB51E748402DBB3C41DC04CFA16C-->  <AD_ORG_ID><![CDATA[0]]></AD_ORG_ID>
+<!--9EEEEB51E748402DBB3C41DC04CFA16C-->  <ISACTIVE><![CDATA[Y]]></ISACTIVE>
+<!--9EEEEB51E748402DBB3C41DC04CFA16C-->  <VALUE><![CDATA[OneMoreSequencesWithDocumentType]]></VALUE>
+<!--9EEEEB51E748402DBB3C41DC04CFA16C-->  <MSGTEXT><![CDATA[There are one or more sequences with the same document type ]]></MSGTEXT>
```

Comment:
have an space in last position

Reply:
It has a space because the getNameOfADSequencesOnConflicts method concatenates the name of the repeated document type.